README.md

```diff
@@ -894,6 +894,22 @@ Service accounts are best as they can be further constrained and not be
 associated with your overall authenticated access. Do not ship your own
 credentials to remote systems.
 
+In order to attach a service account to the instance created by spread use the
+following configuration:
+
+```
+(...)
+
+backends:
+    google:
+        key: $(HOST:echo $GOOGLE_JSON_FILENAME)
+        google-service-account: $(HOST:echo $GOOGLE_SERVICE_ACCOUNT)
```

Comment:
perhaps this could be a boolean switch? The problem is that even through the key type is `service_account`, the actual role associated with the identity of the request may not have the permissions to attach service account to instances. As we learned the hard way, what you need is `compute.instances.setServiceAccount` permission, which wasn't set for spread user role in our GCP project.

Reply:
Alternative I discussed with @sergiocazzolato is to maybe have this as a property of the actualy system eg. `attach-service-account: true`, which then defaults to using the service account `client_email` present in the client authentication file.